src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/SendOrderContestedAboutToSubmitHandler.java

```diff
@@ -137,6 +141,11 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
             resetFields(caseData.getDraftOrdersWrapper());
             clearTemporaryFields(caseData);
             sendOrdersCategoriser.categorise(caseDetails.getData());
+
+            List<DraftOrdersReviewCollection> draftOrdersReviewCollection = draftOrderService
+                .clearEmptyOrdersInDraftOrdersReviewCollection(caseData);
```

Comment:
umm. I think you don't need to return a collection and set it in the handle class. You can simply update it within the `DraftOrderService`.